tests/api.py

```diff
@@ -29,5 +32,15 @@ def test_result_category(self):
     expect_rsp_o["id"] = id
     expect_rsp_o["create_datetime"] = rsp_o["create_datetime"]
     self.assertTrue(rsp_o == expect_rsp_o, "Got unexpected data: {}, expected {}".format(rsp_o, expect_rsp_o))
+
+    #updating a record
+    data2 = {"mobile_number": "123224323", "id" : 1}
+    rsp = self.send_json("/patient/update", data2)
+    print(rsp.json())
+    self.assertTrue(rsp.status_code == 201, "Bad status response code: {}".format(rsp.status_code))
+    rsp_o = rsp.json()
+    print(rsp_o)
+    id = rsp_o["id"]
+    self.assertTrue(rsp_o["msg"] == 'Operation Successful', "Got unexpected msg in response {}".format(rsp_o["msg"]))
 if __name__ == '__main__':
   unittest.main()
```

Comment:
Commenting here since the binary files do not let me put a comment on them. We should not be commiting .pyc files or anything else generated. Do not do git commands with -a option - instead add each file that you mean to add individually, then git commit -m. Another option is to maintain a good .gitignore, but I prefer adding files individually as my approach is - it does not need to be under revision control unless I explicitly told it so. Revision control keeps track of the developer's work, not automated tools, and the developer should be well aware what he or she changed or added.